### DISCUSSION TITLE
First account balance

### DISCUSSION THREAD

**Discussion by nwissac** [firefly-iii/firefly-iii#8627] [Firefly III]
Hi,
I'm just getting started with Firefly III (v 6.1.8) when I set up the first set of bank accounts in the Balance column, I got a warning "The bank balance must be less than or equal 10019092020."  (image attached). I use currency Rp (Indonesian Rupiah).
Please advise how to set up the amount balance.

Thanks

![Warning](https://github.com/firefly-iii/firefly-iii/assets/162290049/3827f42c-977b-485c-8757-77aaf2e08dbd)

**Comment by JC5** [comment]
I believe the error is self-explanatory. There is a maximum amount built in. If you need a higher start balance, create deposits that will help you reach it.
